.github/workflows/prune-dist.yml

```diff
@@ -0,0 +1,39 @@
+# truncate the cockpit-dist history every Sunday night, to avoid unbounded growth
+name: prune-dist
+on:
+  schedule:
+    - cron: '0 1 * * 0'
+  # can be run manually on https://github.com/cockpit-project/cockpit/actions
+  workflow_dispatch:
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up configuration and secrets
+        run: |
+          printf '[user]\n\tname = Cockpit Project\n\temail=cockpituous@gmail.com\n' > ~/.gitconfig
+          echo '${{ secrets.COCKPITUOUS_TOKEN }}' > ~/.config/github-token
+          # we push to -dist repo via https://github.com, that needs our cockpituous token
+          git config --global credential.helper store
+          echo 'https://token:${{ secrets.COCKPITUOUS_TOKEN }}@github.com' >> ~/.git-credentials
+
+      - name: Clone -dist repo
+        run: |
+          git clone https://github.com/${{ github.repository }}-dist.git dist-repo
```

Comment:
Ehm, why don't we run this directly in `https://github.com/${{ github.repository }}-dist.git dist-repo`?
We don't want to set up hooks there?

Reply:
I suppose we could, but I thought it'd be nice to be able to blow away the repo and start from scratch. The publish-dist workflow can deal with an empty repo. But I'm happy to move it to -dist as well.

Reply:
I.e. this is particularly helpful if you work on your cockpit fork -- then you don't have to actually fork the cockpit-dist repo as well (which is usually huge), and just start your own forked -dist from scratch.